Control/Monad/Trans/Control.hs

```diff
@@ -751,6 +752,21 @@ control :: MonadBaseControl b m => (RunInBase m b -> b (StM m a)) -> m a
 control f = liftBaseWith f >>= restoreM
 {-# INLINABLE control #-}
 
+-- | An often used composition: @control f = 'liftBaseWith' f >>= 'restoreM'@
```

Comment:
This refers to `control` instead of `controlT`. Maybe also rephrase the "often used" part because I haven't seen `controlT` much in practise.